codex-ui/dev/pages/components/Popup.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <PageHeader>
+    Popup with Confirm
+    <template #description>
+      A component that appears on top of other components
+    </template>
+  </PageHeader>
+  <Popup>
+    <template #trigger>
+      <Button
+        secondary
+        data-dimensions="large"
+      >
+        Press here to open popup
+      </Button>
+    </template>
+    <template #content>
+      <Confirm
+        title="CodeX"
+        body="Are you sure you want to delete the page?"
+        left-button="Button"
+        right-button="Confirm"
+        :on-left-activate="onLeftFunction"
+        :on-right-activate="onRightFunction"
```

Comment:
confirm should always have the Cancel button and customizable text for the Confirm button